blocklist-client/src/common/mod.rs

```diff
@@ -1,21 +1,37 @@
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 use std::fmt;
-use warp::reject::Reject;
 
-#[derive(Debug)]
-pub enum Error {
-    AddressNotFound,
+pub mod error;
+
+/// The BlocklistStatus of a user address
+#[derive(Serialize, Deserialize)]
+pub struct BlocklistStatus {
+    // Whether the address is blocklisted or not
+    pub is_blocklisted: bool,
+    // The risk severity associated with an address
+    pub severity: String,
+    // Blocklist client's acceptance decision based on the risk severity of the address
+    pub accept: bool,
+    // Reason for the acceptance decision
+    pub reason: String,
 }
```

Comment:
I guess that we might want to reuse this type in the signer so it would be nice to implement some common traits for it:

```suggestion
#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
pub struct BlocklistStatus {
    // Whether the address is blocklisted or not
    pub is_blocklisted: bool,
    // The risk severity associated with an address
    pub severity: String,
    // Blocklist client's acceptance decision based on the risk severity of the address
    pub accept: bool,
    // Reason for the acceptance decision
    pub reason: String,
}
```

Reply:
Also, why is `severity` a string here and not a `RiskSeverity`?